examples/integration_test.py

```diff
@@ -967,4 +1077,11 @@ def print_usage(exitcode, reason=None):
         print('=' * 30, 'Verifying AVRO', '=' * 30)
         verify_avro()
 
+    if 'avro-https' in modes:
```

Comment:
Positional arguments won't work well with multiple modes supported at the same time.
I suggest adding a new `--conf <json>` option that allows you to specify arbitrary configuration settings, in this case the schema.registry SSL settings, and then having an example in the README.